components/block/BlockInfo.tsx

```diff
@@ -1,5 +1,5 @@
 // import BlockHeaderTable from "$/components/BlockHeaderTable.tsx";
-import BlockStampsTable from "$/components/BlockStampsTable.tsx";
```

Comment:
trying to standardize on the $/components type referencing to remove the lint errors and make things consistent